### ISSUE TITLE
demo is broken

### ISSUE THREAD

**Issue by sw-yx** [aweary/react-copy-write#52]
hi brandon!

your demo in the readme: https://codesandbox.io/s/yp34vpk50j is mostly commented out.

i tried to fix it but ran into this weird destructuring issue: https://codesandbox.io/s/qz227r7q0w

can you please take a look? i cant figure it out.

**Comment by sw-yx** [comment]
i made a working todolist demo if that helps https://codesandbox.io/s/nkmow5qx04

**Comment by aweary** [comment]
Hey @sw-yx! Thanks, I must have made edits to it thinking I had forked it 😬

I'll fix this soon.

**Comment by sw-yx** [comment]
yeah for some reason all these online editors make a bad habit of not making it super clear when you are editing the main thing vs editing a fork. only astexplorer.net does it right.

**Comment by dallanlee** [comment]
I would really like to see a working demo. I've been tracking this issue and I'm surprised it hasn't been fixed after so long.
